keyboards/sofle/choc/keymaps/choc_default/keymap.c

```diff
@@ -0,0 +1,366 @@
+ /* Copyright 2022 Brian Low
```

Comment:
Keymap folders need to be renamed to `default` and `via`

Reply:
@fauxpark I did original have the keymaps as `default` and `via` but another contributor noticed that running: 
```
qmk compile -kb sofle/choc -km default
```
would compile the keymap at `sofle/keymaps/default` instead of `sofle/choc/keymaps/default`

Reply:
That's a problem then, if this board doesn't make sense with the higher level keymap. It may need to be a separate board rather than a revision of the sofle.

Reply:
@fauxpark moved into a new board: `/keyboards/sofle_choc`

Opened issue #16829 to coordinate reorganizing existing sofle code. Hopefully with that issue, I can move this board under a `sofle` family top level folder